english/interact/fork/follow/follow.md

```diff
@@ -4,4 +4,5 @@ With the fork still in hand, you pace yourself towards the hopping Mr marshmallo
 
 At one corner of the marshmallow room, you notice a large dark hole and marshmallow just jump straight into it!
 
-Do you:
\ No newline at end of file
+Do you:
+(Ponder the hole?)[ponder/ponder-the-hole.md]
```

Comment:
Did you mean for this to be a link?  If so, your syntax is backwards.  The text should be in square brackets and the link in parentheses rather than vice versa.  You can view the page on GitHub on your fork to check if the link is displaying correctly.